sdks/java/core/src/main/java/org/apache/beam/sdk/coders/CoderRegistry.java

```diff
@@ -535,13 +532,13 @@ public CoderProvider getFallbackCoderProvider() {
     private Coder<?> coder;
     private Type type;
 
-    public IncompatibleCoderException(String message, Coder<?> coder, Type type) {
+    IncompatibleCoderException(String message, Coder<?> coder, Type type) {
```

Comment:
This is fine, but redundant and IMO not quite best practice. It means that if one refactors the class to have a different access level, one also has to scrape through the methods and change them.